config/multiBandDriver.py

```diff
@@ -3,7 +3,7 @@
 import os.path
 
 
-for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd"):
+for sub in ("mergeCoaddDetections", "measureCoaddSources", "mergeCoaddMeasurements", "forcedPhotCoadd", "detectCoaddSources"):
```

Comment:
I may have put it at the front of the list to reflect the order of operation in multiband.